v2/internal/frontend/desktop/darwin/frontend.go

```diff
@@ -39,6 +39,7 @@ var messageBuffer = make(chan string, 100)
 var requestBuffer = make(chan webview.Request, 100)
 var callbackBuffer = make(chan uint, 10)
 var openFilepathBuffer = make(chan string, 100)
+var secondInstanceBuffer = make(chan options.SecondInstanceData, 100)
```

Comment:
Am wondering if this needs to be 100? I imagine there's only normally 1 message getting processed at any time?

Reply:
I'm not Golang expert, so not clearly know how buffered channels works.

Does it mean that if there is 100, then it can handle 100 opening at the same time?

If I will create channel with size 1 – then it will process openings sequentially?

Reply:
Yes, correct. But it's unlikely that you'd ever get more than 1 at the same time. I think setting this to 5 would be fine?

Reply:
Ok, that make sense. Will update values for them